src/sentry/rules/filters/issue_type.py

```diff
@@ -0,0 +1,46 @@
+from collections import OrderedDict
+from typing import Any, Union
+
+from django import forms
+
+from sentry.eventstore.models import Event, GroupEvent
+from sentry.rules import EventState
+from sentry.rules.filters import EventFilter
+from sentry.types.issues import GroupType
+
+TYPE_DESCRIPTIONS = {
+    GroupType.ERROR.value: "Errors",
+    GroupType.PERFORMANCE_N_PLUS_ONE.value: "Performance (N+1)",
+    GroupType.PERFORMANCE_SLOW_SPAN.value: "Performance (slow span)",
+    GroupType.PERFORMANCE_SEQUENTIAL_SLOW_SPANS.value: "Performance (sequential slow spans)",
+    GroupType.PERFORMANCE_LONG_TASK_SPANS.value: "Performance (long task spans)",
+    GroupType.PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN.value: "Performance (render blocking asset span)",
+    GroupType.PERFORMANCE_DUPLICATE_SPANS.value: "Performance (duplicate spans)",
+    GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value: "Performance (N+1 DB queries)",
```

Comment:
chatted with @doralchan about what copy should be. We should stick to the perf issue title to be consistent. We currently define it as "N+1 Query", so the copy here should be `"Performance (N+1 Query)"`
![Screen Shot 2022-10-05 at 2 37 43 PM](https://user-images.githubusercontent.com/23648387/194169035-27fcdf79-0548-479b-a69c-850e657a90b8.png)